Frameworks/UIKit/UIColor.mm

```diff
@@ -226,6 +226,8 @@ @implementation UIColor {
     UIImage* _image;
     id _pattern;
     __CGColorQuad _components;
+    // Note: This should be part of the CGColor struct
+    woc::StrongCF<CGColorSpaceRef> _colorSpace;
```

Comment:
I really don't think this is the way to fix this.  This change set should be included in the separation of CGColor from UIColor, rather than piling on additional short term fixes in UIColor which will add more things to remove.